src/render/draw_raster.js

```diff
@@ -122,7 +122,7 @@ function getFadeValues(tile, parentTile, layer, transform) {
         // if no parent or parent is older, fade in; if parent is younger, fade out
         const fadeIn = !parentTile || Math.abs(parentTile.coord.z - idealZ) > Math.abs(tile.coord.z - idealZ);
 
-        const childOpacity = tile.refreshedUponExpiration ? 1 : util.clamp(fadeIn ? sinceTile : 1 - sinceParent, 0, 1);
```

Comment:
@lbud I was gonna say we should switch this from `!!fadeIn` to `Boolean(fadeIn)`, but realizing now that `fadeIn` is already safely a boolean, based on the line above, so we can just change `!!fadeIn` to `fadeIn`.

Approving, nitpick notwithstanding.